features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -830,6 +830,32 @@ URL - [https://ik.imagekit.io/demo/img/logo-white\_SJwqB4Nfe.png?tr=bg-F00000](h
 
 ![](https://ik.imagekit.io/demo/img/logo-white\_SJwqB4Nfe.png?tr=bg-F00000)
```

Comment:
Need to update tables of supported parameter in overlay or other pages 
https://docs.imagekit.io/features/image-transformations/overlay-using-layers

Reply:
added